common/shlibs

```diff
@@ -3978,3 +3978,7 @@ libsepol.so.1 libsepol-3.0_1
 libfrrcares.so.0 libfrr-7.3.1_1
 libhugetlbfs.so.0 libhugetlbfs-2.22_1
 libdolphinvcs.so.5 dolphin-plugins-20.04.2_1
+libkpathsea.so.6 texlive-20200406_1
+libtexluajit.so.2 texlive-LuaTeX-20200406_1
+libtexlua53.so.5 texlive-LuaTeX-20200406_1
+libptexenc.so.1 texlive-20200406_1
```

Comment:
Btw, adding entries few lines up prevent conflicts with majority of PRs that add at end.